src/GSF.CarbonAware/src/Handlers/CarbonAware/CarbonAwareParameters.cs

```diff
@@ -0,0 +1,378 @@
+using CarbonAware.Model;
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Json.Serialization;
+
+namespace GSF.CarbonAware.Handlers.CarbonAware;
+
+public class CarbonAwareParametersBaseDTO
```

Comment:
Exciting to see this move to the library. Given the story in the board, is the idea to have a duplicate between here and `CarbonAware` project or just have it live here?

Reply:
The long term goal is to have this class in the GSF library and be removed from `Carbonaware` project